### PR TITLE
Add support to use rememberDevice param in polling and verify.

### PR DESCRIPTION
**Pull request by alexeystadnikov-okta**
Resolves OKTA-158993

screen shot 2018-04-27 at 8 13 40 pm
screen shot 2018-04-27 at 8 14 34 pm

@mauriciocastillosilva-okta 
Please, review.

### CODE REVIEW

**Comment by alexeystadnikov-okta** [comment]
Added additional tests for `rememberDevice=true`.

File: package-lock.json [hunk 1]
```diff
@@ -5,9 +5,9 @@
   "requires": true,
   "dependencies": {
     "@okta/okta-auth-js": {
-      "version": "1.13.0",
-      "resolved": "https://registry.npmjs.org/@okta/okta-auth-js/-/okta-auth-js-1.13.0.tgz",
-      "integrity": "sha1-9KX3Oku+MquqULuYWkp0rjnHYkg=",
+      "version": "1.17.0",
+      "resolved": "https://artifacts.aue1d.saasure.com:443/artifactory/api/npm/npm-okta/@okta/okta-auth-js/-/@okta/okta-auth-js-1.17.0.tgz",
```
Comment (mauriciocastillosilva-okta) [inline package-lock.json]: this is wrong, it should be the public npm registry link

Comment (alexeystadnikov-okta) [reply package-lock.json]: Done

**Comment by alexeystadnikov-okta** [comment]
@mauriciocastillosilva-okta 
Travis is Green!

File: package-lock.json [hunk 2]
```diff
@@ -5,9 +5,8 @@
   "requires": true,
   "dependencies": {
     "@okta/okta-auth-js": {
-      "version": "1.13.0",
-      "resolved": "https://registry.npmjs.org/@okta/okta-auth-js/-/okta-auth-js-1.13.0.tgz",
-      "integrity": "sha1-9KX3Oku+MquqULuYWkp0rjnHYkg=",
+      "version": "https://registry.npmjs.org/@okta/okta-auth-js/-/okta-auth-js-1.17.0.tgz",
```
[the remark this answers is not in the document]
Comment (alexeystadnikov-okta) [reply package-lock.json]: I had to install it from the url directly due to the routes to the artifactory, not registry to get the correct sha.
Path to the registry was changed manually.

[the remark this answers is not in the document]
Comment (alexeystadnikov-okta) [reply package-lock.json]: Done using npm

**Pull request merged**